.markdown-linkcheck.json

```
@@ -2,6 +2,9 @@
     "ignorePatterns": [
         {
             "pattern": "classes/grpc.ConcordiumGRPCClient.html"
+        },
+        {
+            "pattern": "https://node.testnet.concordium.com:20000"
```

We are migrating to another endpoint
```suggestion
            "pattern": "https://grpc.testnet.concordium.com:20000"
```